### ISSUE TITLE
Additional comment in the can-observable-object/object.static.props.html

### ISSUE THREAD

**Issue by cherifGsoul** [canjs/canjs#5499]
https://canjs.com/doc/can-observable-object/object.static.props.html#staticprops__PROPERTIES_

<img width="832" alt="props___can-observable-object___Observables___API_Docs___CanJS" src="https://user-images.githubusercontent.com/109013/88549887-c15b4c80-d018-11ea-9304-878ccde986b2.png">

**Comment by phillipskevin** [comment]
I don't think this is an extra comment. I think it just doesn't explain what it is. I think this is describing this case:

![image](https://user-images.githubusercontent.com/5851984/88552143-58a8ab80-cfe9-11ea-9d71-e1d5cada2398.png)
